experimental/cloudstrapper/playbooks/roles/aws-cleanup/vars/main.yaml

```diff
@@ -0,0 +1,57 @@
+---
+deleteStacks:
+  - stackSanRamonAgwANic2
```

Comment:
What are these ?

Reply:
This code cleans up the gateways that were created in the region as part of the Cloudstrapper (https://github.com/magma/magma/pull/5232). I left this code in to illustrate how a Cloudformation based approach might simplify creation, updates and deletion of resources.